src/measure_theory/decomposition/jordan.lean

```diff
@@ -376,6 +376,50 @@ begin
   simp [to_signed_measure_neg],
 end
 
+/-- The total variation of a signed measure. -/
+def to_measure (s : signed_measure α) : measure α :=
```

Comment:
Could you call this `total_variation`? There could be other choices for `to_measure`, for instance the positive part, so removing the ambiguity can only help.